pom.xml

```diff
@@ -1105,8 +1105,8 @@
 			 For instance, our version of Lucene is tied to the version of Hibernate Search. Similarly, our
 			 version of Hibernate Search must be compatible with the version of Hibernate and Hibernate must
 			 be compatible with our version of Spring. -->
-		<springVersion>5.3.5</springVersion>
-		<hibernateVersion>5.4.28.Final</hibernateVersion>
+		<springVersion>5.3.6</springVersion>
```

Comment:
How are these version changes related to the ticket?

Reply:
Sorry I changed them accidentally when I had merge conflicts on the other tickets.

Reply:
I will fix them asap. Anything else?

Reply:
I made them to solve the merge conflicts with main branch. Did I do something wrong?